src/shapes/IText/DraggableTextDelegate.ts

```diff
@@ -140,39 +140,45 @@ export class DraggableTextDelegate {
     };
     target.setSelectionStyles(styleOverride, 0, selectionStart);
     target.setSelectionStyles(styleOverride, selectionEnd, target.text.length);
+    target.dirty = true;
```

Comment:
don't we need to use always `set` for dirty in case is inside an interactive group?

Reply:
well no,
since I need it only for the export I don't want the group to invalidate.
It is a render export matter, as we discussed. All because of this mix we must invalidate the cache so the export is fine...